resources/sass/app.scss

```diff
@@ -239,86 +237,85 @@ dl.import-meta .download-csv {
 }
 
 .button {
-    padding: $wikit-Button-medium-padding-vertical $wikit-Button-medium-padding-horizontal;
-    font-weight: $wikit-Button-font-weight;
-    line-height: $wikit-Button-line-height;
+    @include ui-text-M($font-weight-bold);
+    padding: $spacing-30 $spacing-75;
     display: inline-block;
-    color: $wikit-Button-normal-neutral-color;
-    border-width: $wikit-Button-border-width;
-    border-style: $wikit-Button-border-style;
-    border-color: $wikit-Button-normal-neutral-border-color;
-    border-radius: $wikit-Button-border-radius;
-    background-color: $wikit-Button-normal-neutral-background-color;
-    transition-duration: $wikit-Button-transition-duration;
-    transition-property: $wikit-Button-transition-property;
+    border-width: $border-width-base;
+    border-style: $border-style-base;
+    border-color: $border-color-base;
+    border-radius: $border-radius-base;
+    outline: none;
+    background-color: $background-color-interactive-subtle;
+    transition-duration: $transition-duration-base;
+    transition-property: $transition-property-base;
 
     &:focus {
```

Comment:
based on Codex's button implementation, we should include `outline: @outline-base--focus;` in this state. this apparently is a better alternative to the value `none`. (More info [here](https://gerrit.wikimedia.org/r/plugins/gitiles/design/codex/+/refs/heads/main/packages/codex/src/components/button/Button.vue#184))

Reply:
I think we will do the button outline in a separate PR, this chain is getting too massive